src/common_swagger_api/schema/groups.clj

```diff
@@ -100,13 +102,46 @@
    (describe Boolean "True if the user was added successfully")
 
    :subject_id
-   (describe NonBlankString "The subject ID.")
+   (describe NonBlankString "The subject ID")
```

Comment:
I've been systematically removing periods from schema member descriptions ever since I realized just how funny they look in the Swagger UI.